server/src/routes/api/v1/votesRouter.js

```diff
@@ -0,0 +1,41 @@
+import express from "express"
+import Vote from "../../../models/Vote.js"
+import Review from "../../../models/Review.js"
+
+const votesRouter = new express.Router()
+
+votesRouter.post("/", async (req, res) => {
+  try{
+    if (req.user) {
+      const userId = parseInt(req.user.id)
+      const reviewId = parseInt(req.body.reviewId)
+      const voteStatus = parseInt(req.body.voteStatus)
+      const currentVote = await Vote.query().findOne({ userId, reviewId })
+      const currentReview = await Review.query().findById(reviewId)
+      let newStatus
```

Comment:
do we need to create newStatus, or is there a way we can just use the voteStatus defined on line 12?

Reply:
We could have just used voteStatus, but we liked the idea of using newStatus to grab what was persisted and return that instead.